go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/github/git-sizer
 
-go 1.17
+go 1.23.0
+
+toolchain go1.24.1
 
 require (
-	github.com/cli/safeexec v1.0.0
+	github.com/cli/safeexec v1.0.1
 	github.com/davecgh/go-spew v1.1.1 // indirect
-	github.com/spf13/pflag v1.0.5
-	github.com/stretchr/testify v1.8.1
-	golang.org/x/sync v0.1.0 // indirect
+	github.com/spf13/pflag v1.0.6
+	github.com/stretchr/testify v1.10.0
+	golang.org/x/sync v0.12.0 // indirect
```

Comment:
Are the changes to the `go.mod` and `go.sum` files required for this change?  If so, could you say a few words about why that is?  If not, could they be reverted here?

Reply:
We could undo it, but maintaining an aggressive go.mod policy allows us to fix dependency issues as they arise.